group.py

```diff
@@ -418,8 +418,18 @@ def api_group_create(ctx, group_name, category, subcategory, description, data_c
     :param subcategory:         Subcategory of the group to create
     :param description:         Description of the group to create
     :param data_classification: Data classification of the group to create
+
+    :returns: Dict holding process status and process status info
     """
-    ctx.uuGroupAdd(group_name, category, subcategory, description, data_classification, '', '')
+    try:
+        response = ctx.uuGroupAdd(group_name, category, subcategory, description, data_classification, '', '')['arguments']
+        status = response[5]
+        message = response[6]
+        return {'proc_status': status,
```

Comment:
We should use the API responses instead of custom dicts, example: https://github.com/UtrechtUniversity/yoda-ruleset/blob/b72a51894131b096a95518e826769c013d24c8f7/research.py#L86

Reply:
Gedaan en werkt idd heel mooi!
Het maakt de veranderingen in yoda-portal idd niet meer noodzakelijk.

Gr,
Harm